tests/acceptance/run.sh

```diff
@@ -145,9 +148,7 @@ else
   UNEXPECTED_SUCCESS=false
 fi
 
-if [ "${UNEXPECTED_FAILURE}" = false ] && [ "${UNEXPECTED_SUCCESS}" = false ] && [ "${UNEXPECTED_NIGHTWATCH_CRASH}" = false ]; then
-  FINAL_EXIT_STATUS=0
-else
+if [ "${UNEXPECTED_FAILURE}" != false ] || [ "${UNEXPECTED_SUCCESS}" != false ] || [ "${UNEXPECTED_NIGHTWATCH_CRASH}" = false ] || [ ${FINAL_EXIT_STATUS} -ne 0 ]; then
```

Comment:
```suggestion
if [ "${UNEXPECTED_FAILURE}" = true ] || [ "${UNEXPECTED_SUCCESS}" = true ] || [ "${UNEXPECTED_NIGHTWATCH_CRASH}" = true ] || [ ${FINAL_EXIT_STATUS} -ne 0 ]; then
```

Reply:
`"${UNEXPECTED_NIGHTWATCH_CRASH}" = false` was the problem - all the pipelines are failing.